src/microfrontends/package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "@harness/microfrontends",
-  "version": "1.0.2",
+  "version": "1.0.4",
```

Since this isn't a bug fix, it should be a minor, not a patch.
```suggestion
  "version": "1.1.0",
```

---

This is an update.
We have published packages till 1.0.4 and it has not got updated here.
I am just updating the version